frontend/src/theme/components/Drawer.ts

```diff
@@ -9,22 +10,31 @@ import { textStyles } from '~theme/textStyles'
 
 import { Modal } from './Modal'
 
+const $bg = cssVar('drawer-bg')
+
 const baseStyleOverlay: SystemStyleFunction = (props) => {
   return Modal.baseStyle(props).overlay ?? {}
 }
 
+const baseStyleDialog = defineStyle({
+  [$bg.variable]: 'white',
+})
+
 const baseStyleHeader: SystemStyleObject = {
   ...textStyles['h2'],
   py: '1.25rem',
   px: '2.5rem',
+  bg: 'white',
```

Comment:
to remove, since dialog itself is white already

Reply:
[4c43914](https://github.com/opengovsg/FormSG/pull/7292/commits/4c43914fa37aea5570ca22296e85a96f1f21734b)